validation/src/main/scala/hmda/validation/rules/lar/validity/V285.scala

```diff
@@ -0,0 +1,33 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.census.CBSATractLookup
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+
+object V285 extends EditCheck[LoanApplicationRegister] {
+
+  val cbsaTracts = CBSATractLookup.values
+
+  override def name: String = "V285"
+
+  override def apply(input: LoanApplicationRegister): Result = {
+    val state = input.geography.state
+    val stateCodes = cbsaTracts.map(c => c.state)
```

Comment:
@schbetsy noticed that this could be `cbsaTracts.map(c => c.state).distinct` instead, and could be declared on line 11, outside the `apply` method.  We tried it out, and it did make the tests run noticeably faster (1 second) on the command line.